src/Components/Common/Profile/ProfileDropdown.scss

```diff
@@ -13,4 +16,52 @@
     padding: 0 8px;
     font-size: 0.9em;
   }
+
+  .dropdown-menu {
+    position: absolute;
+  }
+}
+
+#menu {
```

Comment:
@coolbeatz71 Could you also reduce the size of the hamburger menu on mobile display?

Reply:
Ok, let me reduce it